examples/complete/android/demo/src/main/java/com/amazon/styledictionaryexample/color/BaseColorActivity.kt

```diff
@@ -0,0 +1,139 @@
+    ab?.setDisplayHomeAsUpEnabled(true)
+    val recyclerView = findViewById<View>(R.id.base_color_list)!!
+    (recyclerView as RecyclerView).adapter = SimpleItemRecyclerViewAdapter(allBaseColors)
+  }
+
+  private val baseColorList: ArrayList<StyleDictionaryNode>
+    get() {
+      val path = ArrayList<String>()
+      path.add("color")
+      path.add("base")
+      return StyleDictionaryHelper.getArrayAtPath(path)
+    }
+
+  private val allBaseColors: ArrayList<BaseColorListItem>
+    get() {
+      val baseColors = baseColorList
+      val allColors = ArrayList<BaseColorListItem>()
+      val path = ArrayList<String>()
+      path.add("color")
+      path.add("base")
+      for (node in baseColors) {
+        if (node.name != "white" && node.name != "black") {
+          path.add(node.name)
+          path.add("500")
+          val header = StyleDictionaryHelper.getProperty(path)
+          allColors.add(BaseColorHeaderItem(header))
+          path.removeAt(path.size - 1)
+          val props = StyleDictionaryHelper.getArrayOfProps(path)
+          for (prop in props) {
+            allColors.add(BaseColorItem(prop))
+          }
+          path.removeAt(path.size - 1)
+        }
+      }
+      return allColors
+    }
+
+  inner class SimpleItemRecyclerViewAdapter(private val values: List<BaseColorListItem>) :
+    RecyclerView.Adapter<SimpleItemRecyclerViewAdapter.ViewHolder>() {
+    override fun getItemViewType(position: Int): Int {
+      return if (values[position].isHeader) {
+        1
+      } else {
+        0
+      }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+      val view = when (viewType) {
+        0 -> LayoutInflater.from(parent.context)
+          .inflate(R.layout.base_color_list_content, parent, false)
+        1 -> LayoutInflater.from(parent.context)
+          .inflate(R.layout.base_color_list_header, parent, false)
+        else -> LayoutInflater.from(parent.context)
+          .inflate(R.layout.base_color_list_content, parent, false)
+      }
+      return ViewHolder(view)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+      holder.item = values[position]
+      with(values[position]) {
+
+        val id = resources.getIdentifier(property.name, "color", packageName)
+        val fontColor: Int = if (property.attributes["font"] == "inverse") {
+          resources.getColor(R.color.color_font_inverse_base, null)
+        } else {
+          resources.getColor(R.color.color_font_base, null)
+        }
+
+        holder.titleView.text = title
+        holder.titleView.setTextColor(fontColor)
+
+        if (holder.valueView != null) {
+          holder.valueView.text = subtitle
+          holder.valueView.setTextColor(fontColor)
+        }
+
+        holder.view.setBackgroundColor(resources.getColor(id, null))
+        holder.view.setOnClickListener {
+        }
+        //                    Context context = v.getContext();
```

Comment:
Remove these lines? Or it looks like there is a ColorDetailFragment that maybe this was intended to navigate to?

Reply:
These lines were commented out in original files. Didn't want to change too much or add the logic which was not added originally. I can make it in the scope of separate PR. Thanks for your comment!